pkg/cli/values/options.go

```diff
@@ -38,14 +38,12 @@ type Options struct {
 
 // MergeValues merges values from files specified via -f/--values and directly
 // via --set, --set-string, or --set-file, marshaling them to YAML
-func (opts *Options) MergeValues(p getter.Providers) (map[string]interface{}, error) {
+func (opts *Options) MergeValues(p getter.Providers, in io.Reader) (map[string]interface{}, error) {
```

Comment:
why do we need to pass in a reader? `readFile` should only be able to read from `os.Stdin` if the filepath is set to `-`.